turbinia/jobs/plaso.py

```diff
@@ -16,7 +16,7 @@
 
 from __future__ import unicode_literals
 
-from turbinia.evidence import BodyFile
+from turbinia.evidence import BodyFile, DockerContainer
```

Comment:
Tiny nit, but can you put the import on it's own line?  It's something from the style guide:  https://google.github.io/styleguide/pyguide.html#313-imports-formatting